### DISCUSSION TITLE
->model() and ->showQueries() returning 'method not found'

### DISCUSSION THREAD

**Discussion by littlegreenguy** [spatie/laravel-ray#118] [Q&A] [answered]
Hey! working on Ray v.1.9.1, Laravel 8 - when I try either `ray()->model()` or `ray()->showQueries()` functions (obv with data params!) (probably others too, but these are the ones I've tried so far), I get `Method showQueries does not exist.` and ` Method model does not exist.`

Any thoughts/suggestions?

**Comment by freekmurze** [comment, marked as answer]
Make sure to install `spatie/laravel-ray` and not `spatie/ray`

**Reply by littlegreenguy** [reply]
ah! that would be it! lol! always something daft! Thank Freek!

**Reply by ab192130** [reply]
I've installed `spatie/laravel-ray` but it shows `Method showQueries does not exist. `and `Method model does not exist.` anyway. What to do now?
